### ISSUE TITLE
Comments on section 2

### ISSUE THREAD

**Issue by NicoKos** [irtf-nwcrg/draft-irtf-nwcrg-network-coding-satellites#9]
"This section describes the components in satellite system that layson SATCOM systems dedicated to broadband Internet access that follows the DVB standards"

-- This section describes the components of a satellite system that follows the ETSI DVB standards to provide broadband internet access. (Actually, DVB-S standards).

Following the DVB standards has effects on draft scope and terminology. Are you sure you want to do that? The current approach to DVB-S seems to be to pick and choose whatever is convenient to the argument.

A high-level description of a multi-gateway _satellite_ network is provided.

"a bi-directional' - elide a, elide -
Fig 1 doesn't resemble the multi-gateway DVB satellite systems that I have worked on.

Indoor and outdoor unit typically refers to the customer satellite terminal pieces as well, not jut a piece of the gateway. (Indoor - network to/from intermediate frequency IF. Outdoor: IF to/from RF) The satellite link appears to be in the wrong place, or the terminal needs breaking down further. And there's no mention of FECFRAME? PLFRAME is the structure, the data gets turned into FECFRAME. But that has "FEC" in it and this draft aims to downplay FEC... awks. Encaps like GSE or MPE have been skipped, but that's really just added flavour.

**Comment by NicoKos** [comment]
Vincent :
- First sentence is not very clear (structure (that ... that...), missing "a" in "in statellite system that lays...", etc.
- Last sentence "Coding schemes could be applied on both single and aggregated traffic." deserves to be detailed. Will it be discussed later on in this document?

**Comment by NicoKos** [comment]
We have proposed some changes in PR #38. 

On the comment on following DVB standards : 
These documents let us have a common view of the different components of a SATCOM system. If you have any public pointer with other naming, we would be happy to have it to see if there are main issues with the actual naming in the document. 

On the comment on Fig 1, indoor and outdoor unit: 
We think we have clarified the usage of FEC in the document. 
The notion of physical gateway is not actually standardised, but the proposed figure let us explain the different use-cases. This is the main rational of this figure.
